config/settings.yml

```diff
@@ -64,17 +64,6 @@
     :event_catcher:
       :event_catcher_ovirt:
         :poll: 15.seconds
-      :event_catcher_ovirt_network:
```

Comment:
Do we need a data migration to remove these from the settings changes table if they exist?

Reply:
:+1: yeah that is true I'll update the schema PR with this